pages/common/jq.md

```diff
@@ -27,6 +27,10 @@
 
 `cat {{file.json}} | jq 'map(.{{key_name}})'`
 
+- Output the value of multiple keys as a new JSON object (assuming the input JSON has keys `key_name` and `other_key_name`):
```

Comment:
Perhaps this will read better?
```suggestion
- Output the value of multiple keys as a new JSON object (assuming the input JSON has the keys `key_name` and `other_key_name`):
```